app-services/secrets/functions/getsecrets.go

```diff
@@ -114,7 +88,7 @@ func getSecrets(edgexcontext *appcontext.Context, data []getSecretsTestData, sec
 
 	for _, test := range data {
 		edgexcontext.LoggingClient.Info(fmt.Sprintf("------- Test: %v -------", test.testName))
-		secrets, err := edgexcontext.SecretProvider.GetSecrets(test.path, test.keys...)
+		secrets, err := edgexcontext.GetSecrets(test.path, test.keys...)
 		if test.errExpected {
```

Comment:
Remove check for expected error.

Reply:
Also just return the error.

Reply:
We want this to be more like a real word usage, not like a unit test. We are just outputting the secrets rather than using them...

Reply:
Add comment that this is an example and one should never output values of real secrets.